### PR TITLE
Merge 1.x -> main

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

File: lib/puppetserver/ca/action/generate.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ class Generate
   puppetserver ca generate [--help]
   puppetserver ca generate --certname NAME[,NAME] [--config PATH]
                            [--subject-alt-names NAME[,NAME]]
-                           [--ca-client]
+                           [--ca-client] [--force]
```
Comment (justinstoller) [inline lib/puppetserver/ca/action/generate.rb:29]: fwiw, this should be `[--ca-client [--force]]`

**Review by justinstoller — APPROVED** [review]

File: lib/puppetserver/ca/action/generate.rb (hunk 1 above)
Comment (mwaggett) [reply lib/puppetserver/ca/action/generate.rb:29]: ooh, would you mind opening a PR to 1.x with that? Then I can release again and incorporate that into the mergeup.

Comment (justinstoller) [reply lib/puppetserver/ca/action/generate.rb:29]: np, https://github.com/puppetlabs/puppetserver-ca-cli/pull/97

Comment (mwaggett) [reply lib/puppetserver/ca/action/generate.rb]: thank you!

**Pull request merged**